tests/buf_reader.md

```diff
@@ -393,6 +393,80 @@ Exception: Failure "skip_while1".
 - : string = "bbbccc"
 ```
 
+## Big Endian
+```ocaml
+# let i = R.of_flow mock_flow ~max_size:100;;
```

Comment:
Note: the mock flow is used to check that it's pulling the right number of bytes at a time. For fixed-sized reads the output isn't very interesting. You might want to use `R.parse_exn` instead (as in the "Convenience wrapper" section).

Reply:
I'd just do one test for each function, like this:
```
# R.(parse_string_exn BE.uint16) "\128\001" |> Printf.sprintf "0x%x";;
- : string = "0x8001"
```
If it gets that right (using an example with the top-bit set, using all bytes, and not the same LE and BE), it's very unlikely to fail on anything else (and the fuzz tests will check everything anyway). We only really need to check that we read the right number of bytes and didn't confuse signed/unsigned or endianness.

Reply:
Excellent I'll do that, I had forgotten the backslash notation!